packages/accounts/src/kernel-zerodev/e2e-tests/kernel-account.test.ts

```diff
@@ -19,14 +19,14 @@ import {
 import { KernelAccountProvider } from "../provider.js";
 import type { KernelUserOperationCallData } from "../types.js";
 import { KernelBaseValidator, ValidatorMode } from "../validator/base.js";
-import { API_KEY, OWNER_MNEMONIC } from "./constants.js";
+import { RPC_URL, API_KEY, OWNER_MNEMONIC } from "./constants.js";
 import { MockSigner } from "./mocks/mock-signer.js";
 
 describe("Kernel Account Tests", () => {
   //any wallet should work
   const config = {
     chain: polygonMumbai,
-    rpcProvider: `${polygonMumbai.rpcUrls.alchemy.http[0]}/${API_KEY}`,
+    rpcProvider: RPC_URL !== undefined ? RPC_URL : `${polygonMumbai.rpcUrls.alchemy.http[0]}/${API_KEY}`,
```

Comment:
what's the use case for this? 

also nit:
```suggestion
    rpcProvider: RPC_URL != null ? RPC_URL : `${polygonMumbai.rpcUrls.alchemy.http[0]}/${API_KEY}`,
```

Reply:
To be able to run the e2e tests against something other than Alchemy prod

Reply:
Can put this in a separate PR, just needed it for my current testing.

Reply:
all good, we can keep it here